src/components/login/LoginForm.tsx

```diff
@@ -22,13 +17,8 @@ const LoginForm = () => {
   const email = method.watch("email");
   const password = method.watch("password");
 
-  const { data, isPending, isError, mutate } = useMutation<
-    PostLoginResult,
-    Error,
-    PostLoginParams
-  >({
-    mutationFn: postLogin,
-    onSuccess: () => {
+  const { data, isPending, isError, login } = useLogin({
+    onSuccessCallback: () => {
```

Comment:
```suggestion
    onSuccess: () => {
```
이렇게 그대로 사용해도 무방했을 것 같아요!

Reply:
함수 이름을 onSuccess로 해도 의미 전달에 문제가 없을 것 같습니다.
수정하겠습니다!